foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -33,11 +31,15 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
-                        By.CSS_SELECTOR, '.addon-webext-name').text
+                _addon_name_locator = (
+                    By.CSS_SELECTOR, '#addon-webext-perm-header > \
+                                      .addon-webext-name')
```

Comment:
One of these locators should still be on line 22. I would suggest this one, as it should ultimately become the only locator once the changes ride the trains to release and eventually ESR.